src/Compilers/CSharp/Test/Symbol/Symbols/Source/NullablePublicAPITests.cs

```diff
@@ -3815,5 +3815,40 @@ void verify(DefaultExpressionSyntax expr, PublicNullableAnnotation expectedAnnot
                 Assert.Equal(expectedState, info.FlowState);
             }
         }
+
+        [Fact]
+        public void GetTypeInfoOnNullableType()
```

Comment:
>GetTypeInfoOnNullableType [](start = 20, length = 25)

Please add tests for nullable value types.

Reply:
As I've reverted the `NullableTypeSyntax`, I don't think this is relevant anymore.

---
In reply to: [366375766](https://github.com/dotnet/roslyn/pull/40950#discussion_r366375766) [](ancestors = 366375766)

Reply:
> As I've reverted the NullableTypeSyntax, I don't think this is relevant anymore.

Are value types never go through the code paths that are modified in this PR? Are we not interested in what GetTypeInfor returns for them?

---
In reply to: [366588484](https://github.com/dotnet/roslyn/pull/40950#discussion_r366588484) [](ancestors = 366588484,366375766)